sdks/python/apache_beam/examples/snippets/snippets_test.py

```diff
@@ -589,18 +590,30 @@ def test_model_textio_compressed(self):
 
   @unittest.skipIf(datastore_pb2 is None, 'GCP dependencies are not installed')
   def test_model_datastoreio(self):
-    # We cannot test datastoreio functionality in unit tests therefore we limit
+    # We cannot test DatastoreIO functionality in unit tests, therefore we limit
     # ourselves to making sure the pipeline containing Datastore read and write
     # transforms can be built.
     # TODO(vikasrk): Expore using Datastore Emulator.
     snippets.model_datastoreio()
 
   @unittest.skipIf(base_api is None, 'GCP dependencies are not installed')
   def test_model_bigqueryio(self):
-    # We cannot test BigQueryIO functionality in unit tests therefore we limit
+    # We cannot test BigQueryIO functionality in unit tests, therefore we limit
     # ourselves to making sure the pipeline containing BigQuery sources and
     # sinks can be built.
-    snippets.model_bigqueryio()
+    run_locally = False
```

Comment:
<!--new_thread; commit:9d7586c0b982dd7eab8f3e3bbc28864792629dda; resolved:0-->
Why do we have to branch below given that this doesn't change ?

Reply:
To make it easier to test locally, I added a comment clarifying this.